webapp/portlet/src/main/webapp/space-settings/main.js

```diff
@@ -22,7 +22,8 @@ const vuetify = new Vuetify({
 const lang = eXo && eXo.env.portal.language || 'en';
 
 //should expose the locale ressources as REST API
-const url = `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.portlet.Portlets-${lang}.json`;
+const url = [`${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.portlet.Portlets-${lang}.json`,
+  `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.addon.Wallet-${lang}.json`];
```

Comment:
we can not add the wallet bundle here. it is an addon unavailable for Meeds

Reply:
Okay